vita3k/gui/src/install_firmware_dialog.cpp

```diff
@@ -0,0 +1,67 @@
+// Vita3K emulator project
+// Copyright (C) 2018 Vita3K team
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#include <gui/functions.h>
+
+#include "private.h"
+
+#include <host/functions.h>
+#include <host/pup_types.h>
+#include <host/state.h>
+#include <util/log.h>
+
+#include <nfd.h>
+
+namespace gui {
+
+static bool draw_file_dialog = true;
+
+void draw_install_firmware_dialog(GuiState &gui, HostState &host) {
+    nfdchar_t *outpath = nullptr;
+    nfdresult_t result = NFD_CANCEL;
+
+    const float button_width = 60;
+    const float button_height = 20;
```

Comment:
20 is not good here, get button cropped lol, just set like me 
```suggestion
    static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
```

Reply:
I'm not sure how it will be cropped because it looks fine for me, but I agree with setting the height to 0 for default height